src/Compilers/VisualBasic/Portable/Syntax/VisualBasicSyntaxDiffer.vb

```diff
@@ -0,0 +1,27 @@
+﻿Namespace Microsoft.CodeAnalysis.VisualBasic
+    Friend NotInheritable Class VisualBasicSyntaxDiffer
+        Inherits SyntaxDiffer
```

Comment:
seems unfortunate to have a model that requires us to allocate tehse instead of having singletons that can be used in a non-allocating fashion.